common/app/layout/WidthsByBreakpoint.scala

```diff
@@ -195,10 +206,12 @@ object ContentWidths {
       wide =            Some(1920.px))
   }
 
-  def getWidthsFromContentElement(hinting: ContentHinting, relation: ContentRelation): WidthsByBreakpoint = {
+  def getWidthsFromContentElement(contentType: String, hinting: ContentHinting, relation: ContentRelation): WidthsByBreakpoint = {
     relation match {
       case MainMedia => hinting.mainContentWidths
-      case _ => hinting.bodyContentWidths }
+      case BodyMedia if hinting == ContentWidths.Inline && contentType == "LiveBlog" => LiveBlogMedia.Inline
```

Comment:
The breakpoints are different. Compare `LiveBlogMedia.Inline` to `MainMedia.Inline`. Notably, live blogs have a different image size at `leftCol` and `wide`.